dev/charts/charts.html

```diff
@@ -0,0 +1,1359 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Vaadin web components</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <style>
+      body {
+        background: var(--lumo-base-color, #fff);
+      }
+      .theme-switcher {
+        display: flex;
+        padding: 1rem;
+        position: fixed;
+        top: 0;
+        z-index: 1;
+      }
+      .theme-swithcher button {
+        margin-right: .5rem;
+      }
+    </style>
+
+    <div class="theme-switcher">
+      <button onclick="changeTheme('')">Default theme</button>
+      <button onclick="changeTheme('gradient')">Gradient theme</button>
+      <button onclick="changeTheme('monotone')">Monotone theme</button>
+    </div>
+
+    <script>
+      //Crappy theme switcher that doesn't clear the previous theme
```

Comment:
```suggestion
      // Crappy theme switcher that doesn't clear the previous theme
```

Reply:
I moved the theme-switcher to its own component implementation and applied this suggestion there.